datacommons/places.py

```diff
@@ -92,3 +92,59 @@ def get_places_in(dcids, place_type):
   if isinstance(dcids, pd.Series):
     return pd.Series([result[dcid] for dcid in dcids])
   return result
+
+
+def get_pop_obs(dcid):
+  """ Returns :obj:`Place`'s population and observations.
+
+  Args:
+    dcids (:obj:`str`): Dcid of the place.
+
+  Returns:
+    An dictionary of population and observation related to this place.
+
+  Raises:
+    ValueError: If the payload returned by the Data Commons REST API is
+      malformed.
+
+  Examples:
+    We would like to get population and observation of Mountain View
+    `Mountain View <https://browser.datacommons.org/kg?dcid=geoId/0649670>`_.
+
+    >>> get_pop_obs("geoId/0649670")
+    {
+      'name': 'Mountain View',
+      'placeType': 'City',
+      'populations': {
+        'dc/p/013ldrstf6lnf': {
+          'numConstraints': 6,
+          'observations': [
+            {
+              'marginOfError': 119,
+              'measuredProp': 'count',
+              'measuredValue': 225,
+              'measurementMethod': 'CenusACS5yrSurvey',
+              'observationDate': '2014'
+            }, {
+              'marginOfError': 108,
+              'measuredProp': 'count',
+              'measuredValue': 180,
+              'measurementMethod': 'CenusACS5yrSurvey',
+              'observationDate': '2012'
+            }
+          ],
+          'popType': 'Person',
+          'propertyValues': {
+            'age': 'Years16Onwards',
+            'gender': 'Male',
+            'income': 'USDollar30000To34999',
+            'incomeStatus': 'WithIncome',
+            'race': 'USC_HispanicOrLatinoRace',
+            'workExperience': 'USC_NotWorkedFullTime'
+          }
+        }
+      }
+    }
```

Comment:
At the bottom, shall we add/modify this snippet of text?

```
Notice that the return value is a multi-level :obj:`dict`. The top level contains the following keys.

- :code:`name` and  :code:`placeType` provides the name and type of the :obj:`Place` identified by the given :code:`dcid`.
- :code:`populations` maps to a :obj:`dict` containing all :obj:`StatisticalPopulation`'s that have the given :code:`dcid` as its :obj:`location`.

The :code:`populations` dictionary is keyed by the dcid of each :obj:`StatisticalPopulation`. The mapped dictionary contains the following keys.

- :code:`popType` which gives the population type of the :obj:`StatisticalPopulation` identified by the key.
- :code:`numConstraints` which gives the number of constraining properties defined for the identified :obj:`StatisticalPopulation`.
- :code:`propertyValues` which gives a :obj:`dict` mapping a constraining property to its value for the identified :obj:`StatisticalPopulation`. 
- :code:`observations` which gives a list of all :obj:`Observation`'s that have the identified :obj:`StatisticalPopulation` as their :obj:`observedNode`. Each :obj:`Observation` is represented by a :code:`dict` that have the keys: <INSERT AND DOCUMENT ALL KEYS HERE>
```

Reply:
This is very good!

Reply:
Thank you :)